app/helpers/routing_helper.rb

```diff
@@ -2,37 +2,40 @@
 require 'json'
 
 module RoutingHelper
-  def self.seconds_to_minsec(sec)
+  def self.format_seconds_as_minsec(sec)
     format("%<minutes>.2d:%<seconds>.2d", minutes: sec / 60, seconds: sec % 60)
   end
 
-  def self.valid_coordinates(coordinates)
+  def self.valid_coordinates?(coordinates)
     return false if coordinates.blank?
 
-    regex = /^[0-9]{1,2}(\.[0-9]{1,6})?,[0-9]{1,2}(\.[0-9]{1,6})?$/
+    regex = /^-?\d{1,2}(\.\d{1,8})?,-?\d{1,2}(\.\d{1,8})?$/
     coordinates.match(regex)
   end
 
+  def self.routing_url(start, destination)
+    (start_lat, start_long) = start.split(",")
+    (dest_lat, dest_long) = destination.split(",")
+    "http://routing.openstreetmap.de/routed-foot/route/v1/driving/#{start_long},#{start_lat};#{dest_long},#{dest_lat}?overview=full&geometries=geojson"
```

Comment:
This [URL](https://routing.openstreetmap.de/routed-foot/route/v1/driving/52.39318,13.13153;52.39338,13.12973?overview=full&geometries=geojson) does not return a reasonable JSON file. What am I doing wrong? It should return a small path from building A to the canteen.
It returns:
{
	"code": "Ok",
	"waypoints": [
		{
			"hint": "PVkxgQ2eR4GtAAAAAAAAAI8EAAAAAAAAM9V5QQAAAABi89FCAAAAAKEAAAAAAAAAOgQAAAAAAAASbQSAcnwbAzjAugDcdB8DCl_IAAUAjwSdKPv2",
			"distance": 102709.086693,
			"location": [
				52.132978,
				12.238904
			],
			"name": ""
		},
		{
			"hint": "PVkxgQ2eR4GtAAAAAAAAAI8EAAAAAAAAM9V5QQAAAABi89FCAAAAAKEAAAAAAAAAOgQAAAAAAAASbQSAcnwbAzjAugCkdR8DAljIAAUAjwSdKPv2",
			"distance": 102523.662854,
			"location": [
				52.132978,
				12.238904
			],
			"name": ""
		}
	],
	"routes": [
		{
			"legs": [
				{
					"steps": [],
					"weight": 0,
					"distance": 0,
					"summary": "",
					"duration": 0
				}
			],
			"weight_name": "routability",
			"geometry": {
				"coordinates": [
					[
						52.132978,
						12.238904
					],
					[
						52.132978,
						12.238904
					]
				],
				"type": "LineString"
			},
			"weight": 0,
			"distance": 0,
			"duration": 0
		}
	]
}

Reply:
For me it works, as you can see [here](http://localhost:3000/building_map?start=52.39318,13.13153&dest=52.39338,13.12973):
![image](https://user-images.githubusercontent.com/57799997/145618830-12a84e91-88a0-46a4-8c06-7e0e3db1d0e9.png)